components/http-list/OkHttpList.vue

```diff
@@ -189,7 +188,7 @@
 
         items: T[] = [];
 
-        listKey: String;
+        listKey: String = '';
```

Comment:
Why is this change necessary?

Reply:
because the variable is not initialized if it does not have a defined value, it does not exist